client/src/components/UIElements/Button.scss

```diff
@@ -0,0 +1,23 @@
+.button {
+    min-width: 180px;
+    font-weight: 600;
+    height: var(--height-button);
+    padding: 5px;
+    margin: 5px;
+    outline: none;
+    border: none;
+    border-radius: var(--border-radius-button);
+    color: white;
+    background-color: var(--main-blue);
```

Comment:
It should be --main-color, not --main-blue (it might not be blue).
As a rule of thumb, try to think about the meaning of the variable when you name it and not about what value it currently holds.
This rule is true to any variable, not only css variables